presto-main/src/main/java/io/prestosql/connector/informationschema/InformationSchemaMetadata.java

```diff
@@ -310,6 +310,14 @@ private boolean isColumnsEnumeratingTable(InformationSchemaTable table)
                 .map(schema -> new QualifiedTablePrefix(catalogName, schema));
     }
 
+    private <T> Optional<Set<String>> filterEmptyValues(TupleDomain<T> constraint, T column)
```

Comment:
I was thinking of method like `String<Set> removeEmptyValues(Set<String values)`. Then you would use it like `filterString(constraint, column).map(this::removeEmptyValues)`

Reply:
Updated